src/modules/75-cf/components/FlagActivation/ServicesList.tsx

```diff
@@ -97,13 +109,13 @@ const EditServicesModal: FC<EditServicesProps> = ({
 
       {serviceError && <Page.Error message={getErrorMessage(serviceError)} onClick={refetchServices} />}
 
-      {allServices && !serviceError && (
+      {filteredServices && !serviceError && (
         <Layout.Horizontal
           style={{ flexWrap: 'wrap' }}
           flex={{ alignItems: 'center', justifyContent: 'flex-start' }}
           spacing="small"
         >
-          {allServices.map((service, idx) => {
+          {filteredServices?.map((service: ServiceResponseDTO, idx: number) => {
```

Comment:
filteredServices will always have a value here so ya dont need the `?`

```suggestion
          {filteredServices.map((service: ServiceResponseDTO, idx: number) => {
```

Reply:
updated